notebooks/jax_flfm.ipynb

```diff
@@ -38,8 +38,9 @@
     "    return jnp.stack(imgs, axis=0)\n",
     "\n",
     "\n",
-    "image = open_image(\"../data/yale/light_field_image.tif\")\n",
-    "measured_psf = open_image(\"../data/yale/measured_psf.tif\")\n",
+    "# NOTE: The below data files are only present when cloning the repo and not when pip installing the package.\n",
+    "image = open_image(\"tests/data/yale/light_field_image.tif\")\n",
```

Comment:
I think these paths need call the util repo location or be slightly different, e.g.:

`../flfm/data/yale/light_field_image.tif`